.github/workflows/_check-excluded-packages.yml

```diff
@@ -72,13 +73,13 @@ jobs:
           # a version bump
           #
           # cd baby-liminal-extension/
-          # ${CHECK}
```

Comment:
I'm curious, how did it work before

Reply:
Well, it's an exported variable so it worked just fine.  In this change it actually doesn't make too much difference but let's just make this `${{ env.VARNAME }}` syntax default everywhere.  Not in this line, but it's safer that way and there is no need to expand it with double quote.